pom.xml

```diff
@@ -82,7 +82,7 @@
 		<dataintegrityVersion>4.4.3</dataintegrityVersion>
 		<emrapiVersion>1.29.0</emrapiVersion>
 		<eventVersion>2.7.0</eventVersion>
-		<fhirVersion>1.20.0</fhirVersion>
+		<fhir2Version>1.0.0</fhir2Version>
```

Comment:
@slubwama this looks like a downgrade of the module, why are we doing this.

Reply:
@slubwama I have seen the difference.

Reply:
@solemabrothers its not. Its a new module which has  lower versioning numbers